adafruit_wiznet5k/adafruit_wiznet5k_socket.py

```diff
@@ -90,6 +92,19 @@ def gethostbyname(hostname):
     return addr
 
 
+def is_ipv4(host):
+    """Checks if a host string is an IPv4 address.
+    :param str host: host's name or ip
+    """
+    octets = host.split(".", 3)
+    if len(octets) != 4 or not "".join(octets).isdigit():
+        return False
+    for octet in octets:
+        if int(octet) > 255:
```

Comment:
Since IP addresses are passed in by users  please handle a negative value within the octet - check if int(octet <= 0) as well.

Reply:
I may be wrong here (I don't do much Python) but by checking that the octets are only digits using `"".join(octets).isdigit()` I don't have to worry about a negative sign in the string.  Also, in Python 3 a signed integer won't overflow (https://mortada.net/can-integer-operations-overflow-in-python.html). We'll just run out of RAM, in which case, we won't get to line 103 anyway.

Happy to make the change though if I'm missing something or you simply feel it's more clear.

Reply:
No change req'd - I didn't notice the call to `isdigit`.